src/Components/Publishing/__stories__/InvertedCSS.story.tsx

```diff
@@ -1,7 +1,7 @@
 import { Box, color, Flex } from "@artsy/palette"
 import { storiesOf } from "@storybook/react"
 import { unica } from "Assets/Fonts"
-import { NewlyEstablished1 } from "Components/Publishing/EditorialFeature/Components/Vanguard2019/Assets/NewlyEstablished"
+import { NewlyEstablished1 } from "Components/Publishing/EditorialFeature/Components/Vanguard2019/Blobs/NewlyEstablished"
```

Comment:
Noticed some CI issues after merging the SVGs in the `Assets` folder -- I think this may be reserved for the system, so renamed the directory.